com.microsoft.mrtk.core/Subsystems/Hands/IHandsAggregatorSubsystem.cs

```diff
@@ -43,7 +43,7 @@ public interface IHandsAggregatorSubsystem
         /// The playspace-local pose of the grab/pinch location. This is typically
         /// halfway between the thumb tip and the index tip.
         /// </summary>
-        bool TryGetPinchingPoint(XRNode hand, out HandJointPose jointPose);
+        bool TryGetPinchPose(XRNode hand, out HandJointPose jointPose);
```

Comment:
nit: to save on a breaking change, do we _need_ to rename this?

Reply:
no not really.... It was just bugging me 😅

I'll change it back to save on the headaches

Reply:
Done!